skt/kerneltree.py

```diff
@@ -307,6 +327,15 @@ def merge_git_ref(self, uri, ref="master"):
         return (SKT_SUCCESS, head)
 
     def merge_patchwork_patch(self, uri):
+        """
+        Merge the patch from Patchwork run.
```

Comment:
What does "patchwork run" mean? Do you only mean that the patch to be applied lives on Patchwork? In that case, something like "Apply a patch from Patchwork (using `git am`)" would be a better description of the method. If you mean something else, please do elaborate and I can help you figure out the right wording :)